memcached.c

```diff
@@ -3601,6 +3601,31 @@ static void process_watch_command(conn *c, token_t *tokens, const size_t ntokens
     }
 }
 
+static void process_memlimit_command(conn *c, token_t *tokens, const size_t ntokens) {
+    uint32_t memlimit;
+    assert(c != NULL);
```

Comment:
I think the assert() there is cargo culted by me and could be done once at the top of the process_* stack. Tokens can't be null because you can't get to that function if it's null.